python/cog/schema.py

```diff
@@ -50,9 +50,9 @@ class PredictionRequest(PredictionBaseModel):
     output_file_prefix: t.Optional[str]
 
     webhook: t.Optional[pydantic.AnyHttpUrl]
-    webhook_events_filter: t.Optional[
-        t.List[WebhookEvent]
-    ] = WebhookEvent.default_events()
+    webhook_events_filter: t.Optional[t.List[WebhookEvent]] = (
```

Comment:
Changes like this suggest that you don't have your editor set up to do autoformatting the same as `script/format`. Would you look into that?

Reply:
this change is required by one of linters in our make liner: 
`python -m ruff python/cog`
and reformatted by: 
`python -m ruff format python/cog/schema.py`